openreview/venue_request/process/revisionProcess.py

```diff
@@ -194,6 +194,20 @@ def process(client, note, invitation):
             ))
 
         elif invitation_type == 'Submission_Revision_Stage':
+            submission_revision_stage_notes = client.get_references(
+                referent=forum_note.id,
+                invitation='{support_group}/-/Request{number}/Submission_Revision_Stage'.format(
+                    support_group=SUPPORT_GROUP, number=forum_note.number),
+                limit=2
+            )
+            if len(submission_revision_stage_notes) > 1:
+                last_submission_revision_stage_note = submission_revision_stage_notes[-1]
+                expire_revision_stage_name = last_submission_revision_stage_note.content.get('submission_revision_name',
+                                                                                             'Revision')
+            else:
+                expire_revision_stage_name = 'Revision'
+            if expire_revision_stage_name != forum_note.content.get('submission_revision_name', '').strip():
+                conference._Conference__expire_invitations(expire_revision_stage_name)
```

Comment:
where is this method defined?

Reply:
It's the `__expire_invitations` method, this is how python access methods with double leading underscores. https://peps.python.org/pep-0008/#descriptive-naming-styles

Reply:
Wow, I didn't know that!

Reply:
that only issue with this is that the implementation is expiring all the children invitations. I would prefer to change the expiration date of the super invitation only. Could you use `__expire_invitation` instead?